_tips/developing.id.md

```diff
@@ -0,0 +1,638 @@
+{% include box.html type="end" %}
+{:/}
+
+{::options toc_levels="2" /}
+
+{::nomarkdown}
+{% include_cached toc.html type="start" title="Daftar Isi" class="full" %}
+{:/}
+
+-   TOC is created automatically.
+{:toc}
+
+{::nomarkdown}
+{% include_cached toc.html type="end" %}
+{:/}
+
+
+
+## Kaitkan label dengan setiap kontrol pada isian
+
+Gunakan atribut `for` pada elemen `<label>` yang dihubungkan ke atribut `id` dari elemen isiannya, atau gunakan atribut <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr>. Dalam situasi tertentu, menyembunyikan elemen `<label>` secara visual mungkin bisa diterima, namun pada sebagian besar kasus, label diperlukan untuk membantu semua pembaca mengerti input apa yang diperlukan.
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="Contoh: Menggunakan atribut <code>for</code> dan <code>id</code>" class="example" %}
+
+<div class="two-column">
+<figure>
+<figcaption>{% include_cached icon.html name="desktop" %} Hasil penyajian</figcaption>
+<label for="username">Nama pengguna</label>
+<input id="username" type="text" name="username">
+</figure>
+<figure>
+<figcaption>{% include_cached icon.html name="code" %} Penggalan Kode</figcaption>
+{:/}
+
+~~~html
+<label for="username">Nama pengguna</label>
+<input id="username" type="text" name="username">
+~~~
+
+{::nomarkdown}
+</figure>
+</div>
+{% include_cached box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="Informasi Lebih Lanjut" class="simple" %}
+{:/}
+
+* **WCAG**
+  * [Label atau Instruksi 3.3.2](/WAI/WCAG21/quickref/#labels-or-instructions) ([Memahami 3.3.2](/WAI/WCAG21/Understanding/labels-or-instructions))
+* **Tutorial**
+  * [Melabeli Kontrol](/tutorials/forms/labels/)
+
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+## Sediakan alternatif teks untuk gambar
+
+Pastikan alternatif teks untuk gambar ditambahkan ke semua gambar yang bersifat informatif dan fungsional. Gunakan teks alternatif kosong, `alt=""` untuk gambar dekoratif, atau sertakan gambar tersebut pada CSS. Alternatif teks biasanya disediakan oleh mereka yang bertanggung jawab untuk menulis konten.
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="Informasi Lebih Lanjut" class="simple" %}
+{:/}
+
+* **WCAG**
+  * [Konten Nonteks 1.1.1](/WAI/WCAG21/quickref/#non-text-content) ([Memahami 1.1.1](/WAI/WCAG21/Understanding/non-text-content))
+* **Tutorial**
+  * [Gambar](/tutorials/images/)
+* **Kisah Pengguna**
+  * [Menjelaskan nilai dari alternatif teks bagi pengguna penyandang disabilitas netra](/people-use-web/user-stories/#accountant)
+
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+## Identifikasi bahasa halaman dan perubahan bahasa
+
+Tunjukkan bahasa utama setiap halaman dengan menggunakan atribut `lang` pada tag `html`, misalnya `<html lang="en">`. Gunakan atribut `lang` pada elemen jika bahasa elemen tersebut berbeda dari elemen lainnya pada halaman.
```

Comment:
```suggestion
Tunjukkan bahasa utama setiap halaman dengan menggunakan atribut `lang` pada tag `html`, misalnya `<html lang="id">`. Gunakan atribut `lang` pada elemen jika bahasa elemen tersebut berbeda dari elemen lainnya pada halaman.
```

I think it is appropriate for this translation to use this value.

Reply:
@remibetin okay, added this to the next commit.